testsuite/integration-arquillian/tests/pom.xml

```diff
@@ -466,7 +466,7 @@
         <profile>
             <id>auth-server-migration</id>
             <properties>
-                <migration.import.file>src/test/resources/migration-test/migration-realm-${migrated.auth.server.version}.json</migration.import.file>
+                <migration.import.file>target/test-classes/migration-test/migration-realm-${migrated.auth.server.version}.json</migration.import.file>
```

Comment:
Because we may want to filter test resources. For instance, to test migration of drools policy I need to specify the ${project.version} for the policy maven artifact.

Beside that, the same files in ```src/test/resources``` are mirrored to ```target/test-classes```.